ui_framework/components/pager/pager.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { KuiPagerButtons } from './pager_buttons';
+
+export function KuiPager({
+    className,
+    startNumber,
+    endNumber,
+    totalItems,
+    hasPreviousPage,
+    hasNextPage,
```

Comment:
Is there a use case in which `hasPreviousPage` and `hasNextPage` should not be derived from `startNumber`, `endNumber` and `totalItems`? If not, they could be calculated in here as `startNumber > 1` and `endNumber < totalItems` respectively.